test/fuzz/luaL_loadbuffer/serializer.h

```diff
@@ -29,6 +32,7 @@ using namespace lua_grammar;
  * of generated code samples.
  */
 constexpr std::size_t kMaxCounterValue = 5;
+constexpr std::size_t kMaxDepth = 5;
```

Comment:
Please add a comment why `kMaxDepth` is needed. AFAIK it is a max nesting level for loops (?).

Reply:
We decided to drop it